internal/commandhandler/commandhandler.go

```diff
@@ -60,7 +62,7 @@ func NewCommandHandler(id string, responseChan, preprocessingChan chan *ops.Stor
 	cmdWatchSubscriptionChan chan watchmanager.WatchSubscription,
 	parser requestparser.Parser, shardManager *shard.ShardManager, gec chan error,
 	ioThreadReadChan chan []byte, ioThreadWriteChan chan interface{}, ioThreadErrChan chan error,
-	wl wal.AbstractWAL) *BaseCommandHandler {
+	wl wal.AbstractWAL, replay bool) *BaseCommandHandler {
```

Comment:
instead of passing this variable, we can use the `config.Config.EnableWAL` to see if the WAL is enabled. If yes, we can reply and restore. I recommend let's remove this from here, even the argument and use the config right before the restoration.